tests/auth_tests/test_forms.py

```diff
@@ -867,6 +869,29 @@ def test_bug_19349_bound_password_field(self):
         # value to render correctly
         self.assertEqual(form.initial["password"], form["password"].value())
 
+    @override_settings(ROOT_URLCONF="auth_tests.urls_admin")
+    def test_link_to_password_reset_in_helptext(self):
+        user = User.objects.get(username="testclient")
+        form = UserChangeForm(data={}, instance=user)
+        password_help_text = form.fields["password"].help_text
+        matches = re.search('<a href="(.*?)">', password_help_text)
+
+        # Get URL to UserChangeForm in admin via to_field (instead of pk)
+        admin_user_change_url = reverse(
+            "admin:%s_%s_change" % (user._meta.app_label, user._meta.model_name),
+            args=(user.username,),
+        )
+
+        joined_url = urllib.parse.urljoin(admin_user_change_url, matches.group(1))
+
+        pw_change_url = reverse(
+            "admin:%s_%s_password_change"
+            % (user._meta.app_label, user._meta.model_name),
+            args=(user.pk,),
+        )
+        # assert joined_url and pw_change_url are identical
```

Comment:
This comment just needs removing and then the ticket can be marked "ready for checkin" to let felixx know it's ready for him to do a final review.